app/controllers/intents_controller.rb

```diff
@@ -15,4 +17,18 @@ def show
 
     not_found
   end
+
+  private
+
+  def check_uri
+    not_found if uri.blank?
```

Comment:
Currently there is a problem that when accessing `localhost:3000/intent` it raises a error, so 
 handle it